src/main/java/icons/SonarLintIcons.java

```diff
@@ -45,6 +45,8 @@
   public static final Icon WARN = IconLoader.getIcon("/images/warn.png");
   public static final Icon SCM = IconLoader.getIcon("/images/toolWindowChanges.png");
   public static final Icon PROJECT = IconLoader.getIcon("/images/ideaProject.png");
+  public static final Icon NOT_CONNECTED = IconLoader.getIcon("/images/not_connected.svg");
+  public static final Icon CONNECTED = IconLoader.getIcon("/images/connected.svg");
```

Comment:
The SVG icons come from [this list](https://jetbrains.design/intellij/resources/icons_list/). I copied the ones from the "settings sync" plugin; I tried to make a version with a SonarQube or SonarCloud logo, but the icon is very small (16x16 on 1x / 96 DPI) so the logo comes out invisible.